src/components/HomePage.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import MovieCard from "./MovieCard";
+
+import { Container } from "../styles/Container.styled";
+import GlobalStyles from "../styles/Global.styled";
+import { Cards } from "../styles/Cards.styled"
+import PageNotFound from "./PageNotFound";
+import {apiUrl,searchUrl} from "./constants/Global"
+
+
+function HomePage(){
+  const [movies, setMovies] = useState([]);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    getMovies(apiUrl)
+  },
+    [])
+
+  async function getMovies(url) {
+    const response = await axios.get(url);
+    const result = response.data.results;
+
+    setMovies(result);
+  }
+
+  const submit = (e) => {
+    e.preventDefault();
+    if (searchTerm) {
+      getMovies(`${searchUrl}&query=${searchTerm}`)
+    } else {
+      getMovies(apiUrl)
+    }
+  }
+
+  return (
+    <>
+      <GlobalStyles />
```

Comment:
Please add global styles in the index.js file instead of using it in a particular component.